src/System.Net.Requests/pkg/System.Net.Requests.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Net.Requests.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Even before my change, System.Net.Requests does not build for netcore50; try, and it fails with lots of compiler errors.  With bumping the version, anything listed in this line needs to find a matching version of the build that it can include in the package... and since it's a new version, and since there's no build for netcore50, leaving netcore50 here fails. When we need to support this on netcore50 or uap101aot or whatever it's called, the work will need to be done to fix the build and add it back.